src/components/ProductList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Product from './Product.js'
+import products from '../vschoolProducts.js'
+
+function ProductList() {
+
+	const productComponents = products.map(product => <Product key={product.id} name={product.name} 
+		price={product.price.toLocaleString("en-US", { style: 'currency', currency: 'USD' })} description={product.description} />)
+
+	return (	
+			<div>
```

Comment:
Same about code style. Try select all code with ctra+a and ctrl+alt+l (small L) to auto-align code.

Reply:
Will try next time(file was deleted)